src/app/Agent/AgentLiveProbes.tsx

```diff
@@ -0,0 +1,266 @@
+      filtered = direction === SortByDirection.asc ? sorted : sorted.reverse();
+    }
+    setFilteredTemplates([...filtered]);
+  }, [filterText, templates, sortBy]);
+
+  const handleTemplates = React.useCallback(
+    (templates) => {
+      setTemplates(templates);
+      setErrorMessage('');
+      setIsLoading(false);
+    },
+    [setTemplates, setIsLoading, setErrorMessage]
+  );
+
+  const handleError = React.useCallback(
+    (error) => {
+      setErrorMessage(error.message);
+      setIsLoading(false);
+    },
+    [setIsLoading, setErrorMessage]
+  );
+
+  const refreshTemplates = React.useCallback(() => {
+    setIsLoading(true);
+    addSubscription(
+      context.target
+        .target()
+        .pipe(
+          filter((target) => target !== NO_TARGET),
+          first(),
+          concatMap((target) => context.api.getActiveProbes())
+        )
+        .subscribe(
+          (value) => handleTemplates(value),
+          (err) => handleError(err)
+        )
+    );
+  }, [addSubscription, context, context.target, context.api, setIsLoading, handleTemplates, handleError]);
+
+  React.useEffect(() => {
+    addSubscription(
+      context.target.target().subscribe(() => {
+        setFilterText('');
+        refreshTemplates();
+      })
+    );
+  }, [context, context.target, addSubscription, refreshTemplates]);
+
+  React.useEffect(() => {
+    if (!context.settings.autoRefreshEnabled()) {
+      return;
+    }
+    const id = window.setInterval(
+      () => refreshTemplates(),
+      context.settings.autoRefreshPeriod() * context.settings.autoRefreshUnits()
+    );
+    return () => window.clearInterval(id);
+  }, []);
+
+  React.useEffect(() => {
+    const sub = context.target.authFailure().subscribe(() => {
+      setErrorMessage('Auth failure');
+    });
+    return () => sub.unsubscribe();
+  }, [context.target]);
+
+  const displayTemplates = React.useMemo(
+    () => templates.map((t: EventProbe) => [t.id, t.name, t.clazz, t.description, t.methodName + t.methodDescriptor]),
+    [templates]
+  );
+
+  const handleModalToggle = () => {
+    addSubscription(
+      context.api
+        .removeProbes()
+        .pipe(first())
+        .subscribe(() => {
+          refreshTemplates();
+        })
+    );
+  };
+
+  React.useEffect(() => {
+    addSubscription(
+      context.notificationChannel
+        .messages(NotificationCategory.ProbeTemplateApplied)
+        .subscribe((v) => refreshTemplates())
+    );
+  }, [addSubscription, context, context.notificationChannel, setTemplates]);
+
+  const authRetry = React.useCallback(() => {
+    context.target.setAuthRetry();
+  }, [context.target, context.target.setAuthRetry]);
+
+  if (errorMessage != '') {
+    return (
+      <ErrorView
+        title={'Error retrieving events'}
+        message={
+          'This is commonly caused by the agent not being loaded/active, check that the target was started with the agent (-javaagent:/path/to/agent.jar) ' +
```

Comment:
Maybe a period or some other options instead of blank space.

![Screenshot from 2022-10-21 16-33-13](https://user-images.githubusercontent.com/68053619/197283949-ce892733-57b8-47a5-b03b-ef01e30857e1.png)